tf-modules/archive/api.tf

```diff
@@ -10,6 +10,18 @@ resource "aws_cloudwatch_log_group" "api" {
   tags              = local.default_tags
 }
 
+resource "aws_secretsmanager_secret" "api_cmr_password" {
```

Comment:
So we create this resource even if `length(var.cmr_password) == 0`?

Reply:
We do. I don't feel _too_ strongly one way or the other. The actual secret value is set in the `aws_secretsmanager_secret_version` resource, and that's the one that really cares if the password exists or not.